### PR TITLE
Simplify Running the app locally

### PR DESCRIPTION
**Pull request by Jessonsotoventura**
The paths set in the app.py are meant for c9, but I run locally. The paths do not work for me, so I need to change the paths.

I suggest adding a check for the environment variable $LOCAL, which can be TRUE and determines the current working path and inserts it as system path[0] or if any other string is give inserts that into the system path [0]

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -14,8 +14,15 @@
...
-sys.path.insert(0,'/home/ubuntu/workspace/')
-# sys.path.insert(0,'/Users/jesson/Development/Github/Project_ART')
+
+if os.getenv("LOCAL"):
+    if os.getenv("LOCAL").lower() == 'true':
+        sys.path.insert(0,os.getcwd())
+    else:
+        sys.path.insert(0,os.getenv("LOCAL"))
+else:
+    sys.path.insert(0,'/home/ubuntu/workspace/')
```
Comment (memo330179) [inline app.py:24]: If `sys.path.insert(0,os.getcwd())` works locally shouldn't it also work for the c9 enviroment?

Comment (Jessonsotoventura) [reply app.py:24]: Yes it should.
I will remove the else

Comment (Jessonsotoventura) [reply app.py:24]: Oh that line was kept, so that it defaults to the c9 path even when no LOCAL option is passed to python

Comment (Jessonsotoventura) [reply app.py:24]: Also using only `sys.path.insert(0,os.getcwd())` might cause it to fail on an actual server, if the server access it from outside of the git root folder, although I am not sure on that

**Review by memo330179 — APPROVED** [review]
Looks Good

**Pull request merged**